quazip/zip.c

```diff
@@ -540,6 +540,7 @@ local ZPOS64_T zip64local_SearchCentralDir(const zlib_filefunc64_32_def* pzlib_f
 
       if (uPosFound!=0)
         break;
+    }
```

Comment:
Is this brace positioned correctly? I don't get it. As far as I can see, if `uPosFound != 0`, then we already break the inner loop and there is no way we reach this next `if`. Looking at, https://github.com/nmoinvaz/minizip/blob/master/mz_zip.c, I see that it's the way it's intended. Perhaps the only thing we really need to fix here is to add braces for style and clarity *without* changing the logic?